### PR TITLE
Add ENTITIES_SERVICE_ACCESS_TOKEN env var

### LINKED ISSUE #108: 🔐 Support supplying access token to upload
This ensures the CLI can be used in CI/CD workflows, where opening and authorizing a user through a browser window is not possible.

The idea here is to allow passing an access token or similar to the CLI that represents a user, who has already authorized the access for the app on the authorization platform (GitLab @ SINTEF as of the time of writing).

### PR DESCRIPTION
**Pull request by CasperWA**
Closes #108 

This can be set to avoid doing OAuth2 authentication for the CLI, for example when using the CLI in connection with CI/CD.

### CODE REVIEW

File: entities_service/service/security.py [hunk 1]
```diff
@@ -50,6 +53,111 @@ async def get_openid_config() -> OpenIDConfiguration:
         raise ValueError("Could not parse OpenID configuration.") from exc
 
 
+async def verify_user_access_token(token: str) -> tuple[bool, int | None, str | None]:
+    """Verify a user-provided GitLab access token."""
+    # Get current user
+    async with AsyncClient(headers={"Authorization": f"Bearer {token}"}) as client:
```
Comment (Treesarj) [inline entities_service/service/security.py:59]: async with AsyncClient on L94 : suggestion: this request to GitLab API can be reused.

**Review by Treesarj — APPROVED** [review]

File: entities_service/service/security.py (hunk 1 above)
Comment (CasperWA) [reply entities_service/service/security.py:59]: The request is performed within the `async with` block. The two requests are different. I could create the client and reuse it - but I prefer using a context for HTTP requests to ensure any HTTP transports are properly closed.
The alternative is to call the `aclose()` method after usage.

Comment (CasperWA) [reply entities_service/service/security.py:59]: Or maybe I could just initialize it as a variable and then use that in the contexts?

Comment (CasperWA) [reply entities_service/service/security.py:59]: Hmm, after reading more in [the documentation](https://www.python-httpx.org/async/#opening-and-closing-clients), I think maybe going for `aclose()` and a single client instance is the way to go.

**Pull request merged**